src/main/java/com/github/hokkaydo/eplbot/module/code/JavaRunner.java

```diff
@@ -0,0 +1,122 @@
+package com.github.hokkaydo.eplbot.module.code;
+import java.io.File;
+import java.io.IOException;
+import javax.tools.ToolProvider;
+
+import com.github.hokkaydo.eplbot.Strings;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.nio.file.Path;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+public class JavaRunner {
+    private static final String OUTPUT_PATH = System.getProperty("user.dir")+"\\src\\temp\\";
+    public static String run(String input) {
+        if (!input.equals(safeImports(input))){return "Unvalid imports";};
+        String class_name = regexClassName(input);
+        writeFile(input,Path.of(OUTPUT_PATH+"\\"+class_name+".java"));
+        String filePath = OUTPUT_PATH + "\\" + class_name + ".java";
+        try {
+            ByteArrayOutputStream error_stream = new ByteArrayOutputStream();
+            if (ToolProvider.getSystemJavaCompiler().run(null, null, new PrintStream(error_stream), filePath) == 0) {
+                try {
+                    Process process = new ProcessBuilder(System.getProperty("java.home") + File.separator + "bin" + File.separator + "java", "-cp", System.getProperty("java.class.path") + File.pathSeparator + OUTPUT_PATH,regexClassName(input) ).redirectErrorStream(true).start();
+                    Thread timeoutThread = new Thread(() -> {
+                        try {
+                            Thread.sleep(1000*Integer.parseInt(Strings.getString("COMMAND_CODE_TIMELIMIT")));
+                            process.destroy();
+                        } catch (InterruptedException e) {}
```

Comment:
```suggestion
                        } catch (InterruptedException ignored) {}
```
So that my linter understand (he's bit stupid)